examples/bri-3/src/bri/transactions/agents/transactions.agent.spec.ts

```diff
@@ -76,17 +78,49 @@ beforeEach(async () => {
     data: {
       name: 'name',
       description: 'desc',
-      publicKey:
-        '0x047a197a795a747c154dd92b217a048d315ef9ca1bfa9c15bfefe4e02fb338a70af23e7683b565a8dece5104a85ed24a50d791d8c5cb09ee21aabc927c98516539',
+      publicKeys: {
+        createMany: {
+          data: [
+            {
+              type: PublicKeyType.ECDSA,
+              value:
+                '0x047a197a795a747c154dd92b217a048d315ef9ca1bfa9c15bfefe4e02fb338a70af23e7683b565a8dece5104a85ed24a50d791d8c5cb09ee21aabc927c98516539',
+            },
+            {
+              type: PublicKeyType.EDDSA,
+              value:
+                '0x047a197a795a747c154dd92b217a048d315ef9ca1bfa9c15bfefe4e02fb338a70af23e7683b565a8dece5104a85ed24a50d791d8c5cb09ee21aabc927c98516539',
+            },
+          ],
+        },
+      },
+    },
+    include: {
+      publicKeys: true,
     },
   });
 
   bpiSubject2 = await prisma.bpiSubject.create({
     data: {
       name: 'name2',
       description: 'desc2',
-      publicKey:
-        '0x04203db7d27bab8d711acc52479efcfa9d7846e4e176d82389689f95cf06a51818b0b9ab1c2c8d72f1a32e236e6296c91c922a0dc3d0cb9afc269834fc5646b980',
+      publicKeys: {
+        create: [
```

Comment:
@biscuitdey why is this `create` and in L82 it is `createMany`?

Reply:
Fixed.